TestThermo/TestThermo.cs

```diff
@@ -98,54 +98,6 @@ public static void LoadThermoTest2()
             Assert.AreEqual(a.GetAllScansList().First().Polarity, readCovertedMzmlFile.GetAllScansList().First().Polarity);
         }
 
-        [Test]
```

Comment:
Why are these tests gone?

Reply:
ask the bosss

Reply:
@rmillikin

Reply:
these two tests should stay (thermo).

TestPeakTrimmingWithThreeWindows I think has been replaced with another test (TestFilteringPeaksTopN_MultipleWindows) and is ok to delete

Reply:
I'm confused by this answer. stay or go?

Reply:
stay

Reply:
done